test/shared/src/main/scala/zio/test/mock/Capability.scala

```diff
@@ -21,17 +21,20 @@ import java.util.UUID
 import com.github.ghik.silencer.silent
 
 import zio.test.Assertion
-import zio.{ =!=, taggedIsSubtype, taggedTagType, Has, IO, TagType, Tagged, URLayer }
+import zio.{ =!=, taggedIsSubtype, taggedTagType, Has, IO, TagType, Tagged }
 
 /**
- * A `Method[R, I, E, A]` represents a capability of environment `R` that takes an input `I`
+ * A `Capability[R, I, E, A]` represents a capability of environment `R` that takes an input `I`
  * and returns an effect that may fail witn an error `E` or produce a single `A`.
  *
- * To represent polymorphic capabilities you must use one of lazy `Method.Poly` types which
+ * To represent polymorphic capabilities you must use one of lazy `Capability.Poly` types which
  * allow you to delay the declaration of some types to call site.
+ *
+ * To construct capability tags you should start by creating a `Mock[R]` and extend publicly
+ * available `Effect`, `Method`, `Sink` or `Stream` type members.
  */
-abstract class Method[R <: Has[_]: Tagged, I: Tagged, E: Tagged, A: Tagged](val compose: URLayer[Has[Proxy], R])
-    extends Method.Base[R] { self =>
+protected[mock] abstract class Capability[R <: Has[_]: Tagged, I: Tagged, E: Tagged, A: Tagged](val mock: Mock[R])
```

Comment:
Can we add variance here or no because of the tags?

Reply:
We can't. Variance will break things.